
# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50  

# Position Controllers ---------------------------------------
cam_revolute_position_controller:
  type: effort_controllers/JointPositionController
  joint: cam_revolute
  pid: {p: 100.0, i: 0.01, d: 10.0}

cam_prismatic_position_controller:
  type: effort_controllers/JointPositionController
  joint: cam_prismatic
  pid: {p: 100.0, i: 0.01, d: 10.0}
